sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -70,6 +75,10 @@
   private boolean usePaginationReader;
   private List fileLists;
   private Class<? extends CarbonReadSupport> readSupportClass;
+  public static final String UPDATE_DELTA_VERSION = "updateDeltaVersion";
```

Comment:
unused ?

Reply:
it was getting used to check the transactional table, removed now.